internal/cephfs/nodeserver.go

```diff
@@ -228,6 +230,9 @@ func (ns *NodeServer) NodeStageVolume(
 			return nil, status.Error(codes.Internal, err.Error())
 		}
 
+		err = ns.healthChecker.StartChecker(req.GetVolumeId(), stagingTargetPath)
+		log.WarningLog(ctx, "failed to start healthchecker: %v", err)
```

Comment:
this should be done inside `if err !=nil` check

Reply:
Done!